src/Felixkiss/UniqueWithValidator/ValidatorExtension.php

```diff
@@ -34,7 +34,7 @@ public function validateUniqueWith($attribute, $value, $parameters)
         $parameters = array_map('trim', $parameters);
 
         // first item equals table name
-        $table = array_shift($parameters);
+        list($connection, $table) = $this->parseTable(array_shift($parameters));
```

Comment:
Can you add a test for the new behavior? (Asserting that setConnection is called with the correct parameter)